### ISSUE TITLE
[2.15.0] Inserting AllowDynamicProperties attribute breaks final classes

### ISSUE THREAD

**Issue by tminich** [barryvdh/laravel-ide-helper#1516]
### Versions:
- ide-helper Version: 2.15.0
- Laravel Version: 10.44.0
- PHP Version: 8.2.15

### Description:

Apparently 2.15.0 introduced adding the AllowDynamicProperties attribute to generated model helper classes. Unfortunately, if the model class is declared final, this attribute is inserted between the `final` and `class` keywords leading to invalid php.

```
	final #[\AllowDynamicProperties]
	class IdeHelperParticipant {}
```

### Steps To Reproduce:

- Declare a model class final
- Generate model helper (with -M -R, not sure if and how far that matters)

**Comment by barryvdh** [comment]
Does this fix it for you? https://github.com/barryvdh/laravel-ide-helper/pull/1517

**Comment by barryvdh** [comment]
Yeah that fix 

> I guess we just need to shuffle `$keyword` and `$allowDynamicAttributes` in
> 
> https://github.com/barryvdh/laravel-ide-helper/blob/c119e89cf044fd56a98cc92ac0bfc3c8d148b9ea/src/Console/ModelsCommand.php#L1086

Yeah that fixes it, according to the tests

**Comment by tminich** [comment]
Looks like it's fixed in 1.5.1, thanks a lot!
